packages/vuetify/src/components/VSelect/VSelect.tsx

```diff
@@ -55,7 +52,7 @@ export const makeSelectProps = propsFactory({
     default: '$dropdown',
   },
   modelValue: {
-    type: [Number, String, Array],
+    type: [Number, String, Object, Array],
```

Comment:
I think at this point it's just
```suggestion
    type: null,
```